atm.js

```diff
@@ -1,21 +1,192 @@
+// checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//
+// var checkingDeposit = function(amount){
+//   checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//
+// }
+//
+// function checkingWithdrawal(amount){
+//
+// }
+//
+// function savingsDeposit(amount){
+//
+// }
+//
+// function savingsWithdrawal(amount){
+//
+// }
+//
+// // an eventListerner for each button, each one a "click"
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function() {
 
+function checking() {
+  $("#checkingAcct").addClass("zero");
+  this.entries = [];
+  this.els = {
+    // checking value
+    value: $("#checkingAcct"),
+    // input for deposit amount
+    input: $("#checkingAmt"),
+    // checking deposit button
+    checkingDeposit: $("#checkingDeposit"),
+    // checking withdraw button
+    checkingWithdrawal: $("#checkingWithdrawal"),
+  };
 }
 
-function checkingWithdrawal(amount){
+checking.prototype.calculateBal = function() {
+  var total = 0;
+  for (var i = 0; i < this.entries.length; i++) {
+    total += this.entries[i];
+  }
+  // changes color to red if balance hits zero
+  if (total === 0) {
+    $("#checkingAcct").addClass("zero");
+    return total;
+  } else if (total >= 0) {
+    $("#checkingAcct").removeClass("zero");
+    return total;
+  // if userInput > checking balance, set to 0; pull (userInput - checking balance) from savings
+  } else if (total < 0) {
+    $("#checkingAcct").addClass("zero");
+    return 0;
+  }
+};
 
-}
 
-function savingsDeposit(amount){
+//creating a new checking function based on checking constructor function
+var myChecking = new checking();
 
-}
+// what happens when checking deposit button is clicked
+myChecking.els.checkingDeposit.click( function(event) {
+  event.preventDefault();
+  // get user input
+  var userInput = parseInt(myChecking.els.input.val());
+  console.log(userInput);
+  // add the inputs to the array an entries
+  myChecking.entries.push(userInput);
+  // displays user input in checking box
+  myChecking.els.value.html(userInput);
+  // calculate balance
+  var newBal = myChecking.calculateBal();
+  // displays new balance
+  myChecking.els.value.html("$" + newBal);
+});
 
-function savingsWithdrawal(amount){
+// what happens when checking withdrawal button is clicked
+myChecking.els.checkingWithdrawal.click( function(event) {
+  event.preventDefault();
+  // get user input
+  var userInput = parseInt(myChecking.els.input.val());
+  console.log(userInput);
+  // add the negative value of inputs to the array of entries
+  myChecking.entries.push(-(userInput));
+  // displays user input in checking box
+  myChecking.els.value.html(userInput);
+  // calculate balance
+  var newBal = myChecking.calculateBal();
+  // displays new balance
+  myChecking.els.value.html("$" + newBal);
+});
 
+function savings() {
+  $("#savingsAcct").addClass("zero");
```

Comment:
I think it's great how you used constructor functions, but i think you could have used one constructor function for both savings and checkings. So instead of two, just one maybe called account constructor:
```

function Account(type){
  this.name = type;
  this.entries = [];
  this.els = {
    value: $("#" + type + "Acct"),
    input: $("#" + type + "Amt"),
    deposit: $("#" + type + "Deposit"),
    withdrawal: $("#" + type + "Withdrawal"),
  };
}

Account.prototype.calculateBal = function() {
  var total = 0;
  for (var i = 0; i < this.entries.length; i++) {
    total += this.entries[i];
  }
  if (total === 0) {
    $("#" + this.name + "Acct").addClass("zero");
    return total;
  } else if (total >= 0) {
    $("#" + this.name + "Acct").removeClass("zero");
    return total;
  } else if (total < 0) {
    $("#" + this.name +"Acct").addClass("zero");
    return 0;
  }
};

function makeAccounts(){
  var savings = new Account("savings");
  var checking = new Account("checking");
}

makeAccounts();
```